core-web/apps/dotcms-ui/src/app/view/components/dot-copy-button/dot-copy-button.component.html

```diff
@@ -8,6 +8,6 @@
     hideDelay="800"
     tooltipPosition="bottom"
 >
-    <i class="material-icons">content_copy</i>
-    {{ label }}
+    <i class="pi pi-copy"></i>
+    <span class="p-button-label" *ngIf="!!showLabel">{{ label }}</span>
```

Comment:
Isn't the `showLabel` variable already a boolean value?

```suggestion
    <span class="p-button-label" *ngIf="showLabel">{{ label }}</span>
```

Reply:
Yeah my mistake, I meat `*ngIf="!!label"`